docs/supportedsites.md

```diff
@@ -50,6 +50,7 @@
  - **Bandcamp**
  - **Bandcamp:album**
  - **bbc.co.uk**: BBC iPlayer
+ - **bbc.com**: BBC news videos
```

Comment:
This file is updated automatically in releases. Please do not modify it in pull requests.